webpack/assets/javascripts/bundle.js

```diff
@@ -19,6 +19,7 @@ require('./bundle_datatables');
 require('./bundle_lodash');
 
 import compute from './foreman_compute_resource';
+import { jed } from './react_app/common/I18n';
```

Comment:
1. i think you need to import i18n instead of jed? (otherwise tfm.i18n.translate wont work?)
2. should this be inside the react_app or somewhere in this directory? wondering as its not related to react?

Reply:
about the second one, I agree that it doesn't related to react.  however plugins can import foreman files via the `foremanReact` alias, which is the `react_app` folder...